client/src/components/LoginForm/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { useCallback } from 'react';
+import Button from '@material-ui/core/Button';
+
+import { makeRequest } from '../../utils/makeRequest';
+import useInput from '../../hooks/useInput';
+import useApp from '../../hooks/useApp';
+
+import './styles.scss';
+
+const LoginFrom = () => {
+  const [login, loginInput] = useInput({ label: 'Login', className: 'login-form__input', id: 'login-input' });
+  const [password, passwordInput] = useInput({
+    label: 'Password',
+    className: 'login-form__input',
+    type: 'password',
+    id: 'password-input',
+  });
+  const app = useApp();
+
+  const sendRequest = useCallback(async () => {
```

Comment:
Not sure that this method is needed to be wrapped into `useCallback`. Each user action generates a new function. So it's probably just an extra unnecessary memoization.